projects/analog-phytocell/index.md

```diff
@@ -0,0 +1,149 @@
+---
+authors:
+  - "Natalie Schreiber"
+layout: project
+title: "Projekt 1 | Algorithmus | Phytocell"
+type: project
+---
+
+![Cover](splash.png)
+
+# Analog Algorithm | PhytoCell
```

Comment:
The title will be added by the system https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-Projects#markdown-files--syntax